global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "2.1.401"
+    "dotnet": "3.0.100"
```

Comment:
can you update the version in here too to match?
https://github.com/dotnet/roslyn-tools/blob/master/merge-pr-pipeline.yml

There might be a way to pull the version from that file, not sure though.